subsys/spm/spm_ns.c

```diff
@@ -0,0 +1,30 @@
+/*
```

Comment:
This file could instead be placed in subsys/nonsecure, which already has nonsecure parts of SPM functionality.